package.json

```diff
@@ -9,9 +9,9 @@
     "pino-pretty": "./bin.js"
   },
   "scripts": {
-    "ci": "standard && tap 'test/**/*.test.js' --coverage-report=lcovonly && npm run test-types",
+    "ci": "standard && tap --color test/*.test.js test/*/*.test.js --coverage-report=lcovonly && npm run test-types",
     "lint": "standard | snazzy",
-    "test": "tap --100 'test/**/*.test.js'",
+    "test": "tap --100 --color test/*.test.js test/*/*.test.js",
```

Comment:
It was os-dependant and did not work on Windows. I believe current version works the same across platforms.

Reply:
Not sure how it is OS dependent. The point of the quotes is to prevent shell expansion of the pattern so that the Node module `glob` (used internally by `tap`) will do the expansion.

Reply:
It wasn't matching anything on Windows. Actually, we are not even running CI on Windows. Should I restore previous version and enable Windows testing to see what happens?

Reply:
That sounds like a good idea.